packages/typespec-client-generator-core/test/package.test.ts

```diff
@@ -2176,6 +2178,41 @@ describe("typespec-client-generator-core: package", () => {
       strictEqual(apiVersionOpParam.correspondingMethodParams[0], apiVersionClientParam);
     });
 
+    it("global parameters", async () => {
```

Comment:
nit, "subscriptionId client parameter" or "subscriptionId client property" ?

Reply:
This case could cover all the client parameter of ARM sdk, including `credentail/endpoint/subscriptionId/apiVersion`. Maybe `client level signatures` is better.